cmd/client-daemon.go

```diff
@@ -66,6 +68,18 @@ var startDaemonCmd = &cobra.Command{
 			go startProfiler()
 		}
 
+		if os.Getenv("CEDANA_GPU_ENABLED") == "true" {
+			err = pullGPUBinary("gpucontroller", "./gpu-controller")
```

Comment:
let's set filepath for both to /usr/local/bin/cedana-gpu-controller & /usr/local/lib/libcedana-gpu.so